tests/test_CodonBiasComparison.py

```diff
@@ -0,0 +1,247 @@
+"""Pytest for CodonBiasComparison methods in gene_features module."""
+
+import scipy  # type: ignore
```

Comment:
same thing here, please specify the ignore!

Reply:
added!